.github/file-paths.yaml

```diff
@@ -38,10 +38,14 @@ frontend_specs: &frontend_specs
   - "jest.config.json"
   - "jest.tz.unit.conf.json"
 
+release_source: &release_sources
+  - "release/**"
```

Comment:
this will ensure that frontend tests run in ci when we change code in the release directory